strawberry/schema/schema.py

```diff
@@ -26,9 +50,7 @@ def __init__(
         subscription: Optional[Type] = None,
         directives=(),
         types=(),
-        extensions: Sequence[Type[Extension]] = (),
```

Comment:
any reason for removing them from here?

Reply:
Are extensions something the schema should care about? Or should they be at a higher level?

Reply:
Is this the kind of extension that is being dealt with here: https://spec.graphql.org/June2018/#sec-Schema-Extension ?

Reply:
no :) extensions for us are have to add hooks to the schema, so for example to run code when the validation starts or when any resolver runs :)